security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -31,8 +31,10 @@ import (
 
 	"istio.io/istio/pkg/spiffe"
 	"istio.io/istio/security/pkg/listwatch"
-	"istio.io/istio/security/pkg/pki/ca"
+	"istio.io/istio/security/pkg/pki/cainterface"
```

Comment:
Why do you rename it to cainterface?

Reply:
This is to fix the cyclic dependency issue. The interface is moved into cainterface, the implementation is separated. Because the ca implementation now imports controller, and controller package has imported ca already. With this new ca interface, the controller package imports cainterface instead of ca.